Otlp exporter doc update (#4607)
open-telemetry/opentelemetry-dotnet: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

    .Build();
```

> By default, `AddOtlpExporter()` configures the metric exporter as

Reviewer: ```suggestion
By default, `AddOtlpExporter()` pairs the OTLP MetricExporter with a
```

> By default, `AddOtlpExporter()` pairs the OTLP MetricExporter with a

Reviewer: Ideally we just document defaults in the "## Configuration" section for all 3 signals. But that can be a future re-factoring.

Author: Agree. There are some changes coming to logs and traces. Once those are done, we can update the configuration section. I just added this for now as we get common ques around how Otlp metric exporter is configured?

[PeriodicExportingMetricReader](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#periodic-exporting-metricreader)

> with

Reviewer: ```suggestion
with metric export interval of 60 secs and 
```

values of the `PeriodicExportingMetricReaderOptions`
(following the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.12.0/specification/sdk-environment-variables.md#periodic-exporting-metricreader).

| Environment variable                                | `PeriodicExportingMetricReaderOptions` property |
| ----------------------------------------------------| ------------------------------------------------|
| `OTEL_METRIC_EXPORT_INTERVAL`                       | `ExportIntervalMilliseconds`                    |
| `OTEL_METRIC_EXPORT_TIMEOUT`                        | `ExportTimeoutMilliseconds`                     |

> NOTE:  `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` is not supported yet.

Reviewer: nit : add link to the tracking issue.
